app/js/components/CreateEscrow.jsx

```diff
@@ -0,0 +1,82 @@
+import React, {Component} from 'react';
+import {Alert, Button, Card, CardBody, CardHeader, CardTitle, Form, FormGroup, Input, Label} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+function pad(number) {
+  if (number < 10) {
+    return '0' + number;
+  }
+  return number;
+}
+
+function toInputDate(date) {
+  return date.getUTCFullYear() +
+    '-' + pad(date.getUTCMonth() + 1) +
+    '-' + pad(date.getUTCDate()) +
+    'T' + pad(date.getUTCHours()) +
+    ':' + pad(date.getUTCMinutes());
+}
```

Comment:
Shall we extract those 2 functions: pad and toInputDate to utils or something that we can reuse?

Reply:
Good idea